frontend/src/pages/ProfilePage/EditProfileModal.tsx

```diff
@@ -51,35 +109,26 @@ export const EditProfileModal = ({ onClickClose, user: data, refetch }: Props) =
           placeholder="new nickname"
           ref={inputNickRef}
           required
+          onChange={handleChangeNickname}
         />
+        <input type="checkbox" checked={isValidated} readOnly />
         <br />
         <br />
         <label htmlFor="avatar">Edit Avatar</label>
-        <input
-          type="text"
-          id="avatar"
-          defaultValue={data.avatar}
-          placeholder="new Avatar URL"
-          ref={inputAvatarRef}
-          required
-        />
+        <input type="file" id="avatar" onChange={handleChangeImage} />
```

Comment:
```suggestion
        <input type="file" accept="image/*" id="avatar" onChange={handleChangeImage} />
```


<img width="429" alt="스크린샷 2023-02-27 오후 12 46 38" src="https://user-images.githubusercontent.com/73428595/221468855-e854a317-ef7b-409b-ae47-7d5749577515.png">

`accept="image/*"`를 추가하면 사진 처럼 이미지 파일**만** 입력으로 받을 수 있어요.

Reply:
@devleomk1 
![image](https://user-images.githubusercontent.com/37893979/221470880-3984f30c-9a82-456b-9fd7-65e9adcdbb39.png)

Reply:
이런기능이 있다니 몰랐음 감사 !